build.ps1

```diff
@@ -1,10 +1,10 @@
 Param(
     [string] $Configuration = "Debug",
-    [string] $CodeDirectory = ".",
-    [string] $TestsDirectory = ".",
+    [string] $CodeDirectory = "Source",
+    [string] $TestsDirectory = "Source",

```

Comment:
Searching from `.` seemed useless, so I ripped this default up from `appveyor.yml`.